static/js/lib/validation/profile.js

```diff
@@ -263,8 +267,18 @@ const educationKeys: (e: EducationEntry) => string[] = R.ifElse(
   isHighSchool, R.compose(excludeFieldOfStudy, R.keys), R.keys
 );
 
-const educationErrors: (es: EducationEntry[]) => ValidationErrors[] = R.map(entry => (
-  findErrors(entry, educationKeys(entry), educationMessages)
+const schoolLocationIsValid = extraErrorCheck('location', 'Location must contain city, state, country', entry => (
+  isNilOrEmptyString(entry.school_city) ||
+  isNilOrEmptyString(entry.school_state_or_territory) ||
+  isNilOrEmptyString(entry.school_country)
+));
+
+const additionalSchoolValidation = R.converge(mergeListOfArgs, [
+  schoolLocationIsValid
+]);
+
+const educationErrors: (xs: EducationEntry[]) => ValidationErrors[] = R.map(entry => (
+  additionalSchoolValidation(entry, findErrors(entry, educationKeys(entry), educationMessages))
 ));
```

Comment:
could we just define `educationErrors` here using `R.converge`? Something like

```js
const educationErrors = R.converge(
  mergeListOfArgs,
  [
    schoolLocationIsValid,
    R.map(entry => findErrors(entry, educationKeys(entry), educationMessages))
  ]
);
```

might be slightly simpler that way, although the `schoolLocationIsValid` function would have to just return an empty object when there were no errors.